blocks/gallery-block/features/gallery/default.jsx

```diff
@@ -1,35 +1,65 @@
-import React from 'react';
+import React, { lazy, Suspense } from 'react';
 import PropTypes from 'prop-types';
-import { useFusionContext } from 'fusion:context';
+import { useContent } from 'fusion:content';
+import { useFusionContext, useAppContext } from 'fusion:context';
 import getProperties from 'fusion:properties';
 import getTranslatedPhrases from 'fusion:intl';
-import GlobalContentGallery from './_children/global-content';
-import CustomContentGallery from './_children/custom-content';
+
+import { Gallery, ErrorBoundary } from '@wpmedia/engine-theme-sdk';
+
+const AdFeature = lazy(/* istanbul ignore next */ () => import('@wpmedia/ads-block'));
 
 const GalleryFeature = (
   {
     customFields: {
       inheritGlobalContent,
       galleryContentConfig,
     } = {},
-  } = {},
+  },
```

Comment:
was removing the default deliberate? if not, 

```suggestion
  } = {},
```

Reply:
There is no need to have this default